pulley/src/opcode.rs

```diff
@@ -77,7 +74,9 @@ macro_rules! define_extended_opcode {
 
         impl ExtendedOpcode {
             /// The value of the maximum defined extended opcode.
-            pub const MAX: u16 = define_opcode!( @max $( $name )* ) + 1;
+            pub const MAX: u16 = $(
+                if true { 1 } else { ExtendedOpcode::$name as u16 } +
+            )* 0;
```

Comment:
Leftover from debugging or something?

Reply:
Ah no this was intentional. The recursive definition required upping `#[recursion_limit]` which I'd prefer to not do. In lieu of fancier `macro_rules!` features which will make this nicer in the future I opted to have a chain of `1 + 1 + ...` for each opcode name. That requires using `$name` in the repetition somewhere though so I came up with this funky way of always evaluating to `1` but still using `$name` somehow.